### PR TITLE
feat: Hotel page

### LINKED ISSUE #8: Booking: App (Hotels)
Should get a list of hotels, that has been requested from the api.

Design: https://www.figma.com/file/sl27ZKzB6GB7hkxxlvjXaY/Overnites?node-id=48%3A141&t=1PzB3ddIAo5LYWlS-0
Depends on: HF5-PROJECT/api#18

### PR DESCRIPTION
**Pull request by frederikpyt**
## Description
1. Added the hotel page
2. Added the Roboto font and set it as the default font 

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/hotel/view/hotel.dart [hunk 1]
```diff
@@ -9,12 +14,97 @@ class HotelPage extends StatefulWidget {
...
+        }
+
+        for (final hotel in snapshot.data!) {
+          widgets.add(HotelCard(hotel: hotel));
+        }
+
+        if (widgets.isEmpty) {
+          widgets.add(
+            const Text(
+              'Der er ingen hoteller, prøv at refresh...',
```
Comment (Anders164a) [inline lib/hotel/view/hotel.dart]: English?

**Review by Anders164a — CHANGES_REQUESTED** [review]

**Comment by kristianbinau** [comment]
Doesn't handle long descriptions that good

**Comment by kristianbinau** [comment]
Doesn't handle changing already inserted cards

File: lib/hotel/widgets/hotel_card.dart [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                  style: TextStyle(
+                    fontSize: 32,
+                    color: Theme.of(context).colorScheme.onPrimary,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+                SizedBox.fromSize(
+                  size: const Size.fromHeight(4),
+                ),
+                TextWidget(
```
Comment (kristianbinau) [inline lib/hotel/widgets/hotel_card.dart:57]: Doesn't handle long descriptions that good

File: lib/hotel/widgets/hotel_card.dart [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  const HotelCard({super.key, required this.hotel});
+
+  final Hotel hotel;
+
+  @override
+  State<HotelCard> createState() => _HotelCardState();
+}
+
+class _HotelCardState extends State<HotelCard> {
+  late final Hotel hotel;
```
Comment (kristianbinau) [inline lib/hotel/widgets/hotel_card.dart]: Doesn't handle changing already inserted cards

**Review by kristianbinau — CHANGES_REQUESTED** [review]

File: lib/hotel/view/hotel.dart (hunk 1 above)
Comment (frederikpyt) [reply lib/hotel/view/hotel.dart]: Danish?

File: lib/hotel/widgets/hotel_card.dart (hunk 3 above)
Comment (frederikpyt) [reply lib/hotel/widgets/hotel_card.dart]: Fixed

File: lib/hotel/widgets/hotel_card.dart (hunk 2 above)
Comment (frederikpyt) [reply lib/hotel/widgets/hotel_card.dart:57]: Fixed

File: lib/hotel/view/hotel.dart (hunk 1 above)
Comment (Anders164a) [reply lib/hotel/view/hotel.dart]: Change to English

**Review by Anders164a — APPROVED** [review]
?No hablas español¿

**Review by kristianbinau — APPROVED** [review]

**Pull request merged**